src/Reflection/TokenStream.php

```diff
@@ -14,38 +14,38 @@ class TokenStream
      * @internal
```

Comment:
If these constants are now private, it is probably no longer required to keep the @internal annotation. 
This annotation is used to denote that the publically available function is only used within the package itself, and shouldn't be used outside of it. http://docs.phpdoc.org/references/phpdoc/tags/internal.html 
Now that these constants are private, it is no longer possible to use them outside the package anyway.

Reply:
:+1: